docs/lib/restapi/fragments/js/cancel.md

```diff
@@ -0,0 +1,35 @@
+## CANCEL API requests

```

Comment:
Do we need two of the same files to be created? We can reuse the `cancel-request` fragment within this similar documentation. Let me know what you think

Reply:
I was thinking about it, right now I have two different examples `API.get` and `API.graphql` in the two files. Also was wondering if cross referencing fragments was a good idea. If you feel it's better to reuse, I can make the change.